### DISCUSSION TITLE
FP 1.12.2_08: how to set the RAM?

### DISCUSSION THREAD

**Discussion by Illegit** [freeplane/freeplane#1909] [Help] [answered]
Hi with the new release, ther is a difference to set the RAM:
If I need to set it to 2GB how to do it?

> -XX:MaxRAM=20g -XX:MaxRAMPercentage=15.0 

Thanks

**Comment by dpolivaev** [comment]
How did you set the RAM before and in which file?

**Comment by Illegit** [comment]
![1719933973003](https://github.com/freeplane/freeplane/assets/62674157/1f8a15e6-f1f5-4f4e-9797-9329317deaab)
 see above the change in freeplane.bat, and then update accordingly freeplane.l4j.ini and freeplaneConsole.l4j.ini. how do you suggest to rewrite them?

**Comment by dpolivaev** [comment, marked as answer]
Whenever you find a combination of -XX:MaxRAM=20g and -XX:MaxRAMPercentage=15.0, you can replace it by -Xmx2g or -Xmx8g depending on your needs

**Reply by Illegit** [reply]
OK. Thanks.
